ibis/expr/tests/test_table.py

```diff
@@ -953,6 +953,23 @@ def test_join_overlapping_column_names(table):
     assert_equal(joined, expected)
 
 
+def test_join_overlapping_column_names_multiple_join(table):
+    t1 = ibis.table([('foo', 'string'),
+                     ('bar', 'string'),
+                     ('value1', 'double')])
+    t2 = ibis.table([('foo', 'string'),
+                     ('value2', 'double')])
+    t3 = ibis.table([('bar', 'string'),
+                     ('value3', 'double')])
+
+    joined = t1.join(t2, ['foo']).join(t3, ['bar'])
+
+    expected = t1.join(t2, t1.foo == t2.foo).materialize()
```

Comment:
Please add tests (or parameterize) for the basic join types: inner, left, right and outer.